ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import time
+
+logger = logging.getLogger("testrunner")
+
+
+def test_reboot_master(deployment, platform, skuba, kubectl):
+    # Make sure there is at least two masters
+    assert skuba.num_of_nodes('master') > 1
+
+    logger.info('Wait for all the nodes to be ready')
+    kubectl.run_kubectl('wait --for=condition=Ready node --all --timeout=5m')
+
+    logger.info('Rebooting master 1')
+    platform.ssh_run('master', 1, 'sudo reboot')
+
+    # Give the reboot a bit to start
+    time.sleep(5)
```

Comment:
I thin using a fixed time will introduce too much flakynes. We could use the same approach than it used in terraform and retry:
```
for retry in range(10):
     try: 
           platform.ssh_run( 'master', 1 , '/usr/bin/true'):
           break   
     except:
           time.sleep(5)
```

Reply:
Maybe this is not needed as we are waiting for the kubectl as long as we have more than one master ...